sqlserver/assets/configuration/spec.yaml

```diff
@@ -135,6 +135,28 @@ files:
       value:
         type: integer
         example: 5
+    - name: custom_queries
+      description: |
+        Get metrics from a custom SQL query but only if the database is writable
+        (i.e. it's the master in an availability group) Notes:
+        - Custom SQL query must be defined in its own instance
+        - For performance counter queries, use `custom_metrics` instead
+        See XXX for more information.
```

Comment:
Can delete this until we get a blog post up, our `postgres` check includes a similar doc reference.

Reply:
Hey Mike! I noticed this is in the same format as the other db integrations' custom queries. 

Could we use the `instances/db` template here for consistency?